### PR TITLE
Keep values in form

### PR DESCRIPTION
**Pull request by SimonLab**
- Display user values in form when error occurs
- Create default value to dwyl/labels for source repository

ref: #111

### CODE REVIEW

File: lib/labels_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -130,5 +146,14 @@ defmodule LabelsWeb.PageController do
     end
   end
 
+  defp form_values(opts \\ []) do
```
Comment (SimonLab) [inline lib/labels_web/controllers/page_controller.ex:149]: My first thought was to create a new Ecto Schema for the form with virtual fields as we don't want to save directly the form fields to Postgres. Then using a changeset to display the value to the form, however it seems a bit too complicated and ended up just using a keyword list for the form values that we pass to the view.

File: lib/labels_web/controllers/page_controller.ex [hunk 2]
```diff
@@ -33,14 +35,28 @@ defmodule LabelsWeb.PageController do
...
              repo_name: target_repo
            }) do
+      # create ok changeset -> keep source info, remove target info
       conn
       |> put_flash(:info, "Labels synched!")
-      |> redirect(to: Routes.page_path(conn, :index))
+      |> render(
+        "index.html",
+        [repositories: repositories] ++
+          form_values(source_owner: source_owner, source_repo: source_repo)
```
Comment (SimonLab) [inline lib/labels_web/controllers/page_controller.ex:44]: When sync is ok, we only want to keep the source owner/repo in the form and clear the target infos.

File: lib/labels_web/controllers/page_controller.ex [hunk 3]
```diff
@@ -33,14 +35,28 @@ defmodule LabelsWeb.PageController do
...
-        |> redirect(to: Routes.page_path(conn, :index))
+        |> render(
+          "index.html",
+          [repositories: repositories] ++
+            form_values(
+              source_owner: source_owner,
+              source_repo: source_repo,
+              target_owner: target_owner,
+              target_repo: target_repo
+            )
```
Comment (SimonLab) [inline lib/labels_web/controllers/page_controller.ex:58]: When there is an error (ie target or source repo not found) we keep all the values entered by the user in the form

**Review by nelsonic — APPROVED** [review]
@SimonLab looks great. 👍

**Pull request merged**